modules/RoyalCommands/src/main/resources/config.yml

```diff
@@ -689,3 +689,25 @@ items:
       # The lore to apply to spawned items. This allows for easy identification without /spawninfo.
       lore:
       - "&9Spawned"
+
+###
+# Death Message Options #
+###
+death:
+  # "use_custom_death" - Should custom death messages be used?
+  use_custom_death: true
+  # "show_interworld" - Should death messages from other worlds be displayed to players?
+  show_interworld: true
+  # "disabled_worlds" - List of worlds to disable death messages in
+  disabled_worlds:
+    - someworldhere
```

Comment:
Hmm. Based on the code, I don't think this disables death messages overall, just disables _custom_ death messages in that world, it'll still show the vanilla death message (or one from another plugin). We could rename this to `ignored_worlds` ("List of worlds to leave death messages untouched`) and add a `silenced_worlds` ("List of worlds to never show death messages").

Reply:
As suggested there is now `ignored_worlds` and `silenced_worlds`, this was done in 7889235
`ignored_worlds` for when vanilla messages are to be used
`silenced_worlds` for when no messages are required